tests/build.rs

```diff
@@ -5,31 +5,24 @@ pub mod fixture;
 
 use std::process::Command;
 use std::str;
-use std::sync::Mutex;
 
 use assert_cmd::prelude::*;
-use fixture::Fixture;
-
-lazy_static! {
-    static ref BUILD_LOCK: Mutex<u8> = Mutex::new(0);
-}
+use fixture::{Fixture, WranglerToml};
 
 #[test]
 fn it_builds_webpack() {
-    let fixture = Fixture::new("webpack");
```

Comment:
fixtures no longer use name for their directory name, but instead just accept whatever random directory the OS passes to them.